scripts/library.sh

```diff
@@ -200,6 +212,18 @@ function get_app_pods() {
   kubectl get pods ${namespace} --selector=app=$1 --output=jsonpath="{.items[*].metadata.name}"
 }
 
+# Dumps pod logs for the given app.
+# Parameters: $1 - app name.
+#             $2 - namespace.
+function dump_app_logs() {
+  echo ">>> Knative $(capitalize ${REPO_NAME//-/}) $1 logs:"
```

Comment:
There are now 2 places using this approach (here and release.sh). Create a readonly env var called something like "REPO_NAME_FORMATTED" in library.sh and use it.

Reply:
Done.